selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/debugger/DebuggerWatchExpressionTest.java

```diff
@@ -71,7 +77,10 @@
   @BeforeClass
   public void prepare() throws Exception {
     URL resource = getClass().getResource(PROJECT_PATH);
-    prjServiceClient.importProject(ws.getId(), get(resource.toURI()), PROJECT, PLAIN_JAVA);
+    Path sourceFolder = get(resource.toURI());
```

Comment:
Can we create `.project` and `.classpath` files inside `TestProjectServiceClient.importProject` method in case if project is plain java project?

Reply:
done